server/tests/unit/common/test_api.py

```diff
@@ -720,33 +717,86 @@ def test_dataset_does_not_exist(self):
         response = self.client.get(url)
         self.assertEqual(response.status_code, 404)
 
+
+class TestDatasetMetadata(BaseTest):
+
+    @classmethod
     @patch("server.data_common.dataset_metadata.requests.get")
-    def test_config_with_portal_metadata(self, mock_get):
-        response_body = json.dumps(
-            {
-                "collection_id": "4f098ff4-4a12-446b-a841-91ba3d8e3fa6",
-                "collection_visibility": "PUBLIC",
-                "dataset_id": "2fa37b10-ab4d-49c9-97a8-b4b3d80bf939",
-                "s3_uri": f"{FIXTURES_ROOT}/pbmc3k.cxg",
-                "tombstoned": "False",
-            }
+    def setUpClass(cls, mock_get):
```

Comment:
Can you add a test where the portal response fails to ensure this fails gracefully/the explorer is still accessible if the portal is down?

Reply:
Sure, that's a good idea.